src/components/hooks/useElementPosition.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Custom hook that returns an element reference and the position of the
+ * element it refers to.
+ *
```

Comment:
add a jsdoc description for the parameter

it should be the ref (like useElementDimensions) not only the dom element